test/sequential/test-debugger-launch.mjs

```diff
@@ -0,0 +1,35 @@
+import { skipIfInspectorDisabled } from '../common/index.mjs';
+skipIfInspectorDisabled();
+
+import { path } from '../common/fixtures.mjs';
+import startCLI from '../common/debugger.js';
+
+import assert from 'assert';
+const script = path('debugger', 'three-lines.js');
+const cli = startCLI([script]);
+try {
+  await cli.waitForInitialBreak();
+  await cli.waitForPrompt();
+  assert.match(cli.output, /debug>/, 'prints a prompt');
+  assert.match(
+    cli.output,
+    /< Debugger listening on [^\n]*9229/,
+    'forwards child output'
+  );
```

Comment:
This is identical to the regular expression in the existing test. I'm OK with changing it, but I'd prefer that happen in a separate PR. At least two other debugger tests have checks that include `<` and it would probably be best to have that as a single separate change.